kitsune/products/utils.py

```diff
@@ -0,0 +1,21 @@
+from django.db.models import Q
+
+from kitsune.products.models import Banner
+
+
+def cta_banner(products=[], topics=[], page_type="", request=None):
+    query = Banner.objects
+
+    if products:
```

Comment:
I suspect that the following queries will change after the changes in the schema so I will go through them after the update.